src/main/java/frc/robot/subsystems/AimingSubsystem.java

```diff
@@ -0,0 +1,283 @@
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import com.playingwithfusion.TimeOfFlight;
+import com.playingwithfusion.TimeOfFlight.RangingMode;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkBase.IdleMode;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.ElevatorFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj.motorcontrol.Talon;
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import edu.wpi.first.wpilibj2.command.PrintCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.constants.AimingConstants;
+import frc.robot.constants.CompConstants;
+import frc.robot.constants.AimingConstants.AimState;
+
+
+public class AimingSubsystem extends SubsystemBase {
+  public enum AimingMotorMode {COAST, BRAKE}
+
+  // Elevator Hardware
+  private final TalonFX mLeftElevatorMotor = new TalonFX(AimingConstants.LEFT_ELEVATOR_TALON_ID,"rio");
+  private final TalonFX mRightElevatorMotor = new TalonFX(AimingConstants.RIGHT_ELEVATOR_TALON_ID,"rio");
+  
+  // TOF
+  private final TimeOfFlight mElevatorTOF = new TimeOfFlight(AimingConstants.ELEVATOR_TOF_ID);
+  
+  // Wrist Hardware
+  private final CANSparkMax mLeftWristMotor = new CANSparkMax(AimingConstants.LEFT_WRIST_SPARK_ID, MotorType.kBrushless);
+  private final CANSparkMax mRightWristMotor = new CANSparkMax(AimingConstants.RIGHT_WRIST_SPARK_ID, MotorType.kBrushless);
+  private final DutyCycleEncoder mWristThroughBoreEncoder = new DutyCycleEncoder(AimingConstants.ELEVATOR_THROUGHBORE_ENCODER_ID);
+
+  // Current States
+  private double mCurrentElevatorDistanceIn = AimingConstants.MIN_ELEVATOR_DIST_IN;
+  private double mCurrentWristRotationDeg = AimingConstants.MIN_WRIST_ROTATION_DEG;
+
+  // Desired States
+  private double mDesiredElevatorDistanceIn = mCurrentElevatorDistanceIn;
+  private double mDesiredWristRotationDeg = mCurrentWristRotationDeg;
+
+  // MotorMode Chooser
+  private final SendableChooser<AimingMotorMode> mChooser = new SendableChooser<>();
+
+  // PID Controllers and Motor Configs 
+  PIDController mElevatorController = AimingConstants.mElevatorPIDConstants.toWPIController();  
+  PIDController mWristController = AimingConstants.mWristPIDConstants.toWPIController();
+
+  ArmFeedforward mWristFeedforwardController = new ArmFeedforward(AimingConstants.mWristPIDConstants.mKS, AimingConstants.mWristkG, AimingConstants.mWristPIDConstants.mKS);
+  ElevatorFeedforward mElevatorFeedforward = new ElevatorFeedforward(AimingConstants.mElevatorPIDConstants.mKS, AimingConstants.mElevatorkG, AimingConstants.mElevatorPIDConstants.mKV);
```

Comment:
mElevatorFeedForwardController or mWristFeedforward. Make it consistent

Reply:
Fixed now.